Procurement/Controls/FragmentStash.xaml

```diff
@@ -0,0 +1,87 @@
+﻿<UserControl x:Class="Procurement.Controls.FragmentStash"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:local="clr-namespace:Procurement.Controls"
+             mc:Ignorable="d"
+             d:DesignHeight="592" d:DesignWidth="591" Height="592" Width="591"
+             d:DataContext="{d:DesignInstance local:FragmentStashViewModel}">
+    <UserControl.Resources>
+        <Style x:Key="SingleCellItemStyle" TargetType="local:ItemDisplay">
+            <Setter Property="Width" Value="40" />
+            <Setter Property="Height" Value="40" />
+        </Style>
+    </UserControl.Resources>
+    <Canvas Margin="0,-8,0,8">
```

Comment:
Remove the `Margin` attribute here.  It's making the background image sit too far up (not in line with how the other tabs are drawn, and overlapping (underlapping?) the tab names above it).

Reply:
Done